### PR TITLE
configure for codebuild pr builds

### PR DESCRIPTION
**Pull request by jesterhazy**
## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have updated the [changelog](https://github.com/aws/sagemaker-python-sdk/blob/master/CHANGELOG.rst) with a description of my changes (if appropriate)
- [ ] I have updated any necessary [documentation](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by jesterhazy** [comment]
do not merge until the backend codebuild changes are deployed.

File: tox.ini [hunk 1]
```diff
@@ -4,13 +4,10 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = py27,py35,flake8,pylint,twine,sphinx
+envlist = flake8,pylint,twine,sphinx,py27,py36
```
Comment (laurenyu) [inline tox.ini]: we should update the README to reflect this

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=7b3f4cf5-05b7-40b1-b30a-280ecd9c8224%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
### AWS CodeBuild CI Report

* Result: FAILED
* [Build Logs](https://9dzx6wxcy9.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=39d4024e-5632-48ce-b6b2-a806a6695a35%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by jesterhazy** [comment]
overriding test failures - one is transient tf segfault, two are permissions errors on a kms key (fixed in iam).

**Pull request merged**